docs/kubernetes/cross-cluster.md

```diff
@@ -0,0 +1,229 @@
+
+    ```console
+    $ kubectl get pvc
+    NAME          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
+    test-pvc-fs   Bound    pvc-88959393-01c7-415f-aac4-a974c45f7ec7   1Ti        RWX            csi-filestore   4m17s
+    ```
+
+    The PersistentVolume is created with a ReclaimPolicy `Retain`.
+    ```console
+    $ k get pv pvc-88959393-01c7-415f-aac4-a974c45f7ec7
+    NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                 STORAGECLASS    REASON   AGE
+    pvc-88959393-01c7-415f-aac4-a974c45f7ec7   1Ti        RWX            Retain           Bound    default/test-pvc-fs   csi-filestore            2m37s
+    ```
+
+    From `clusterA`, ensure the pods are up and running
+    ```console
+    $ kubectl get pod
+    NAMESPACE                  NAME                                                      READY   STATUS    RESTARTS   AGE
+    default                    web-server-deployment-cluster-a-6fc54d4776-7lzkb          1/1     Running   0          3m3s
+    default                    web-server-deployment-cluster-a-6fc54d4776-nspm7          1/1     Running   0          3m3s
+    default                    web-server-deployment-cluster-a-6fc54d4776-qdttl          1/1     Running   0          3m3s
+    ```
+3. Capture the filestore instance details i.e the filestore instance name, share name from `VolumeHandle` and the filestore instance IP from `VolumeAttributes`.
+    ```console
+    $ kubectl describe pv pvc-88959393-01c7-415f-aac4-a974c45f7ec7
+    ```
+
+    ```yaml
+    Name:            pvc-88959393-01c7-415f-aac4-a974c45f7ec7
+    Labels:          <none>
+    Annotations:     pv.kubernetes.io/provisioned-by: filestore.csi.storage.gke.io
+    Finalizers:      [kubernetes.io/pv-protection]
+    StorageClass:    csi-filestore
+    Status:          Bound
+    Claim:           default/test-pvc-fs
+    Reclaim Policy:  Retain
+    Access Modes:    RWX
+    VolumeMode:      Filesystem
+    Capacity:        1Ti
+    Node Affinity:   <none>
+    Message:         
+    Source:
+        Type:              CSI (a Container Storage Interface (CSI) volume source)
+        Driver:            filestore.csi.storage.gke.io
+        FSType:            ext4
+        VolumeHandle:      modeInstance/us-central1-c/pvc-88959393-01c7-415f-aac4-a974c45f7ec7/vol1
+        ReadOnly:          false
+        VolumeAttributes:      ip=10.159.102.90
+                            storage.kubernetes.io/csiProvisionerIdentity=1608006582064-8081-filestore.csi.storage.gke.io
+                            volume=vol1
+    Events:                <none>
+    ```
+
+3. Switch context to the second cluster `ClusterB` and create a pre-provisioned PersistentVolume resource pointing to the filestore instance
+
+    ```console
+    $ kubectl apply -f examples/kubernetes/cross-cluster/preprov-pv-clusterB.yaml
```

Comment:
Potentially worth adding a note in the README to edit this file before applying it with values captured above (even though it's pretty clear in the yaml file itself - up to you).

Reply:
Added